### DISCUSSION TITLE
Need help Custom room welcome

### DISCUSSION THREAD

**Discussion by Megidius** [mozilla/hubs#4664] [Hubs Cloud Questions]
Hello,
I'm starting developp a room for one event, I would like to custom the welcome page with the name of my factory
could you indicate me how to do it? Is it possible to put only join the room ?

How can I switch off the chat?
Thanks for help

**Comment by vvdt** [comment]
For customizing the look and feel check this: https://hubs.mozilla.com/docs/hubs-cloud-customizing-look-and-feel.html

For turning of the chat, you will need to use a custom client and disable it in the code as far as I know.

**Comment by Megidius** [comment]
Thanks for help

How can i remove the ‘objets’ box
***@***.***

My parameters don’t allow people to use this part

***@***.***

***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jsn3XcwLSJUZihtGUriMPWPLO6sFUzCYF2bsIfB2xZe_5cPcZaCBR_irvkShuJgT4S7dGs5U_QhJQL_wIq9rLRO>

Mickael EGIDIUS
Marketing Produits
T. +33 1 64 61 68 68 | M. +33 6 33 45 30 70
19 rue Albert Einstein - 77420 Champs-sur-Marne - France
www.factorysystemes.fr<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jtqQWzypW0CO8KeBu8jaUBE4wkQJeF48pXZNjd-u1Qe5PA38zq1wvwtV8CXxVCvU-9WbnDSsTwBNdAOeLynt08w>    |   Contact Support/SAV<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jtqQWzypW0CO8KeBu8jaUBE4wkQJeF48pXZNjd-u1Qe5GEbcar5cI3eRS4nQ4vSEK14N53YLah39b--FpYXBpuji77Ydb52Qoa0Rn2kPzC9rxzQFZ0hHuYzCU-6fC699rnd5yO3RhpUxyDQFGmUJi1n>
***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jtqQWzypW0CO8KeBu8jaUBEmc0TKTjSXZXdEmspBtjlBGkO3v7u-cm3GPA9fKBBeJwr-u37aj9516oXKuhWhha-6KKWT2stBRX9b5rC7qQFeHhYYD-KqBQb-eJLq2Jik6ygvCrYjiX6jkXkJhaBlMSmoY6u0rzTt7yS6OP-0WAo3g==>
***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jt8IgpHantCRF2n4okAhYgFzVmez7zghkbdm2pN5fwiofkafCfQ8gUQTSW7A4g2ondWbnDSsTwBNdAOeLynt08w>
***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jthevsk2sAue6I_eefuSR-IliUzHXDQx9O0z2sIS5HWxd5XflVC_6wYXiPolnJvaK3M6ZCm8pUUOl7ogYkLBmeuO0v4CqxcGYjSGO89eGZGJg==>
***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jvrryBMWQeBZnYXOCT0qJUVALXclEnmKzaqzhuqE8PASY_5lm8Tp-e2xrsOuUYbX-qm3A48uOzi_wS6KVgD_5VZzOmQpvKVFDpe6IGJCwZnrjtL-AqsXBmI0hjvPXhmRiY=>
***@***.***<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5juZ6EyhDFb8gAydjUdDcdTyhDms_Kprai22_8lY4BvZy3cH1UqyXw0iUBe_bARr7S3M6ZCm8pUUOl7ogYkLBmeuO0v4CqxcGYjSGO89eGZGJg==>
Factory Systemes SAS est une filiale de Factory Systemes Groupe SAS - www.factorygroupe.fr<https://cloud.letsignit.com/collect/bc/5df4eecb9a37a00009efae4c?p=sn7qbGPjjX-QFLVwn8zgKoHKiRA6WQq-BR_ynyuF7Tq03hU5L_7Q_vbMCfG1AVHw0r-uTuUEWD6ALRWHO232vcA_pDUIQnZVCF8DtNYV5jsn3XcwLSJUZihtGUriMPWPCYMvPyP7C-qUjifS3RuFlXLME5WvOzEufN1L50stGGlW67AVz5hQok-huyUxtmvu>



De : Vincent van den Tol ***@***.***
Envoyé : mercredi 29 septembre 2021 15:19
À : mozilla/hubs
Cc : Mickael EGIDIUS; Author
Objet : Re: [mozilla/hubs] Need help Custom room welcome (Discussion #4664)


For customizing the look and feel check this: https://hubs.mozilla.com/docs/hubs-cloud-customizing-look-and-feel.html

For turning of the chat, you will need to use a custom client and disable it in the code as far as I know.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/mozilla/hubs/discussions/4664#discussioncomment-1402498>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AVYI3VOYLQCELK5ERKMPIODUEMG5XANCNFSM5ETQY7WQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by vvdt** [reply]
You also need to do that by adjusting the custom client code
